bombastic/index/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+pub mod packages;
+pub mod sbom;
```

Comment:
I think the rest of this file should be moved to sbom.rs and removed from lib.rs